flow-typed/storybook.flow.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+declare module "@storybook/react" {
+    /**
+     * Made up type to decorate a React component with the story field.
+     */
+    declare type StoryComponentType<T> = React$ComponentType<*> & {
+        story?: $FlowFixMe,
+        ...
+    };
+}
```

Comment:
Couldn't add flow-typed typelib for this module and I checked its types; it doesn't cater to this specific situation. So I hand-crafted something simple to suit our needs/usage.

I could be persuaded to make `<*>` be `<Empty>` since all current uses seem to be that way.